pkg/serverutils/utils.go

```diff
@@ -0,0 +1,33 @@
+package serverutils
+
+import (
+	"encoding/json"
+	"github.com/coreos/pkg/capnslog"
+	"net/http"
+)
+
+var (
+	plog = capnslog.NewPackageLogger("github.com/openshift/console", "serverutils")
+)
+
+// Copied from Server package to maintain error response consistency
```

Comment:
A follow, we can eliminate this comment since its no longer `copied`.  This is the only occurance.